catalystwan/models/configuration/feature_profile/sdwan/system/device_access_ipv6.py

```diff
@@ -0,0 +1,147 @@
+
+from catalystwan.api.configuration_groups.parcel import Default, Global, Variable, _ParcelBase
+from catalystwan.models.configuration.feature_profile.common import RefIdItem
+
+BaseAction = Literal[
+    "accept",
+    "drop",
+]
+DestinationPort = Literal[161, 22]
+
+
+class SourceDataPrefix(BaseModel):
+    model_config = ConfigDict(extra="forbid", populate_by_name=True)
+
+    source_data_prefix_list: RefIdItem = Field(
+        validation_alias="sourceDataPrefixList", serialization_alias="sourceDataPrefixList"
+    )
+
+
+class SourceIPPrefix(BaseModel):
+    model_config = ConfigDict(extra="forbid", populate_by_name=True)
+
+    source_ip_prefix_list: Union[Global[List[IPv6Interface]], Variable] = Field(
+        validation_alias="sourceIpPrefixList", serialization_alias="sourceIpPrefixList"
+    )
+
+
+class DestinationDataPrefix(BaseModel):
+    model_config = ConfigDict(extra="forbid", populate_by_name=True)
+
+    destination_data_prefix_list: RefIdItem = Field(
+        validation_alias="destinationDataPrefixList", serialization_alias="destinationDataPrefixList"
+    )
+
+
+class DestinationIPPrefix(BaseModel):
+    model_config = ConfigDict(extra="forbid", populate_by_name=True)
+
+    destination_ip_prefix_list: Union[Global[List[IPv6Interface]], Variable] = Field(
+        validation_alias="destinationIpPrefixList", serialization_alias="destinationIpPrefixList"
+    )
+
+
+class MatchEntries(BaseModel):
+    model_config = ConfigDict(extra="forbid", populate_by_name=True)
+
+    destination_data_prefix: Optional[Union[DestinationIPPrefix, DestinationDataPrefix]] = Field(
+        default=None, validation_alias="destinationDataPrefix", serialization_alias="destinationDataPrefix"
+    )
+    destination_port: Global[DestinationPort] = Field(
+        validation_alias="destinationPort", serialization_alias="destinationPort"
+    )
+    source_data_prefix: Optional[Union[SourceIPPrefix, SourceDataPrefix]] = Field(
+        default=None, validation_alias="sourceDataPrefix", serialization_alias="sourceDataPrefix"
+    )
+    source_ports: Optional[Global[List[int]]] = Field(
+        default=None, validation_alias="sourcePorts", serialization_alias="sourcePorts"
+    )
+
+
+class Sequences(BaseModel):
+    model_config = ConfigDict(extra="forbid", populate_by_name=True)
+
+    base_action: Union[Global[BaseAction], Default[Literal[BaseAction]]] = Field(
+        default=Default[Literal[BaseAction]](value="accept"),
+        validation_alias="baseAction",
+        serialization_alias="baseAction",
+    )
+    match_entries: MatchEntries = Field(
+        validation_alias="matchEntries", serialization_alias="matchEntries", description="Define match conditions"
+    )
+    sequence_id: Global[int] = Field(validation_alias="sequenceId", serialization_alias="sequenceId")
+    sequence_name: Global[str] = Field(validation_alias="sequenceName", serialization_alias="sequenceName")
+
+
+class DeviceAccessIPv6Parcel(_ParcelBase):
```

Comment:
Can we modify build helpers so it is more consistent with other models in policies?
I believe this could be easier for user if he can define this item as follows:
```python
dap = DeviceAccessIPv6Parcel(…)
sequence = dap.add_sequence(…, destination_port="22")
sequence.match_source_data_prefix("{{var}}")
sequence.match_source_data_prefix([IPv6Interface(…)]) 
```